scripts/distro.sh

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env bash
+# Infer Unix-like distribution
+#
+# TODO: This logic appears verbatim in a few places.
+# Deduplicate by having other scripts call this one.
```

Comment:
Can you dump this in a function in `utils.sh` (same directory) and source that? That's where we keep reusable functionality.

Reply:
Yes, of course.